elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -988,6 +988,8 @@ elytron.distributed-realm.remove=The remove operation for the security realm.
 # Attributes
 elytron.distributed-realm.realms=References to one or more security realms for authentication and authorization.
 elytron.distributed-realm.realms.realm=Name of the referenced realm.
+elytron.distributed-realm.ignore-unavailable-realms=Whether subsequent realms should be checked after an unavailable realm is reached. The default value is false.
```

Comment:
This is imprecise. It should be clear that the authn process fails by default when an unavailable realm is reached and this attribute is false.

Reply:
I suggest to also clarify that an exception is thrown.

The descriptions in the XSD should be also updated.